### ISSUE TITLE
Inspired by this...

### ISSUE THREAD

**Issue by peterot** [rionshin/HomeKitInfluxGrafana#4]
Hey,

I came across this and was inspired to have another go at writing something to pull the temperature info from the smart devices in my flat. In the end I wrote something to pull as much info as possible from all my HomeKit devices and am writing it straight to InfluxDB. 

It requires building the app with Xcode which is a bit of a shame but it does collect a lot of interesting data. 

Sharing here in case it's of interest - https://github.com/peterot/HomekitDataExporter

Thanks for pulling all this together.

**Comment by rionshin** [comment]
thank you so much for the kind words ! I am not real developer but I will check your repo and come back to you if I need some help .
Can I run this app on MacOS or it need to be on iOS ? 

> Hey,
> 
> I came across this and was inspired to have another go at writing something to pull the temperature info from the smart devices in my flat. In the end I wrote something to pull as much info as possible from all my HomeKit devices and am writing it straight to InfluxDB.
> 
> It requires building the app with Xcode which is a bit of a shame but it does collect a lot of interesting data.
> 
> Sharing here in case it's of interest - https://github.com/peterot/HomekitDataExporter
> 
> Thanks for pulling all this together.

**Comment by peterot** [comment]
Yeah, you should be able to build and run on a Mac. Do let me know how you get on :)
